fastexcel-writer/src/test/java/org/dhatim/fastexcel/CorrectnessTest.java

```diff
@@ -309,6 +309,7 @@ void testForAllFeatures() throws Exception {
 
             //set values for cell
             ws.value(0, 0, "Hello fastexcel");
+            ws.value(0, 1, "Hello fastexcel (inline)", true);
```

Comment:
I'm not sure about this API. Reading this line, it's hard to figure out what does the `true` stand for.

Reply:
I admit it.
Which would be better?

1. `ws.inlineString(0, 1, "Hello fastexcel (inline)");`
2. `ws.value(0, 1, "Hello fastexcel (inline)", StringStoreType.INLINE);`
3. `ws.value(0, 1, new InlineString("Hello fastexcel (inline)"));`

Or, could you suggest something else?

Reply:
+1 for `ws.inlineString`

Reply:
@ochedru Thank you for your comment!

I changed it to `ws.inlineString`.